objects/obj_creation_popup/main.js

```
@@ -0,0 +1,719 @@
+        <div className="text-xl font-semibold tracking-tight">Floor Plan Sketcher</div>
+        <div className="ml-auto flex items-center gap-2">
+          <label className="px-3 py-2 rounded-2xl bg-gray-100 hover:bg-gray-200 cursor-pointer">
+            <input type="file" accept="image/*" className="hidden" onChange={(e)=> e.target.files && onImportImage(e.target.files[0])} />
+            Import Plan Image
+          </label>
+          <label className="px-3 py-2 rounded-2xl bg-gray-100 hover:bg-gray-200 cursor-pointer">
+            <input type="file" accept="application/json" className="hidden" onChange={(e)=> e.target.files && importJSON(e.target.files[0])} />
+            Import JSON
+          </label>
+          <button onClick={exportPNG} className="px-3 py-2 rounded-2xl bg-blue-600 text-white hover:bg-blue-700 shadow">Export PNG</button>
+          <button onClick={exportJSON} className="px-3 py-2 rounded-2xl bg-blue-50 text-blue-700 hover:bg-blue-100">Export JSON</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)